package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@axa-ch/patterns-library",
+  "name": "@axa-ch/pattern-library",
```

Comment:
shouldn't it be `@axa-ch/patterns-library`?

Reply:
the currently published version of the PL is 1.XX and not ours, so it doesn't really matter. For consistency I changed the name here. I does not habe an impact anyways.